### DISCUSSION TITLE
Export settings and subscriptions

### DISCUSSION THREAD

**Discussion by akwala** [bitfireAT/icsx5#69] [Ideas]
Use case: Replicating the app settings and subscriptions on a different/new device.

Export (and import) the app settings and subscriptions, including stored credentials.

Encrypt the exported data with a password – prompt for it in export workflow – that is required for importing the data.

**Comment by Stunkymonkey** [comment]
Easy solution: use the OPML format.
Encrypting will make it much more complicate. IMHO: if someone wants to encrypt it they can take care themself.

**Comment by ArnyminerZ** [comment]
Currently the native Android Auto Backup is supported (#119), and was released in [v2.1](https://github.com/bitfireAT/icsx5/releases/tag/v2.1).

I think a manual backup export can be useful, but if we follow that "if someone wants to encrypt it they can take care themself" maybe the best export option would be SQLite, since subscriptions are currently being stored that way. However, OPML can also be an option, or simply JSON format. @rfc2822 can say which option might be the best.

Maybe we should convert this into an issue, and keep track of it there.
